File: birthday/submissions/accepted/birthdayBrute.cpp
#include <iostream>
#include <math.h>  //For the pow function
using namespace std;

/**
 * TOOD: 
 * 
 * A: a non-negative integer
 */
int solve(int A) {
    // YOUR CODE HERE
    int count = 0;
    for(int i =1; i < A; i++){
        count += std::pow(i,2);
    }
    return count +3;
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        int A;
        cin >> A;
        cout << solve(A) << '\n';
    }
}

File: birthday/submissions/accepted/birthdayBrute.py
def solve(A: int) -> int:
    """
    TODO: 
    
    A: a non-negative integer
    """
    # YOUR CODE HERE
    numDays =0
    for year in range(1,A + 1):
        numDays += year**2
    return numDays +3


def main():
    T = int(input())
    for _ in range(T):
        temp = input().split()
        A = int(temp[0])
        print(solve(A))

if __name__ == '__main__':
    main()
